CSharpFunctionalExtensions/Result/Error/Error.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace CSharpFunctionalExtensions
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+
+    [DebuggerDisplay("{Message}")]
+    public sealed class Error : ValueObject
+    {
+        private const string Separator = "||";
+
+        public string Code { get; }
+        public string Message { get; }
+
+        internal Error(string code, string message)
+        {
+            Code = code;
+            Message = message;
+        }
+
+        protected override IEnumerable<object> GetEqualityComponents()
+        {
+            yield return Code;
+        }
+
+        public string Serialize()
+        {
+            return $"{Code}{Separator}{Message}";
+        }
+
+        public static Error Deserialize(string serialized)
+        {
+            if (serialized == "A non-empty request body is required.")
```

Comment:
this seems like custom code that shouldn't be in this library

Reply:
Class deleted